soufi/finder.py

```diff
@@ -30,6 +30,7 @@ class SourceType(enum.Enum):
     java = "java"
     go = "go"
     nuget = "nuget"
+    crate = "crate"
```

Comment:
I'm torn on this one; we call Python sources `python` even though the file format is `egg` or `wheel`, Java sources `java` even though the file format is `jar`, and we call Ruby sources `gem`; so I guess `crate` is following prior art, but prior art is a goddamn mess.

Reply:
Same, I deliberated a while over this myself. In the end, I decided it's probably more prudent to separate the packaging format from the language. We already did that with yum really.